LINUX.md

```diff
@@ -0,0 +1,177 @@
+# TerraMA² - Build and Generate Package in Linux
+
+## Dependencies that require manual installation on Linux
+
+First of all make sure that your machine has the following installed requirements, before proceeding:
+* Git (call in shell command line):
+```
+$ sudo apt-get install git
+```
+* Build-Essentials (call in shell command line):
+```
+$ sudo apt-get install build-essential
+```
+* [CMake (preferably version 3.0.2)](https://cmake.org/download/) or call in shell command line:
+```
+$ sudo apt-get install cmake-gui
+```
+
+## Build and Install TerraLib
+
+You should build and install TerraLib package required by TerraMA², according to the link below:
+ - [Build and install TerraLib Package](http://www.dpi.inpe.br/terralib5/wiki/doku.php?id=wiki:documentation:devguide#downloading_the_source_code_and_building_instructions).
+
+## Cloning TerraMA² Repository
+
+1.1. Open the shell command line.
+
+1.2. Make a new folder to host TerraMA² source code:
+```
+$ mkdir -p /home/USER/mydevel/terrama2/codebase
+```
+
+1.3. Change the current directory to that new folder:
+```
+$ cd /home/USER/mydevel/terrama2/codebase
+```
+
+1.4. Make a local copy of TerraMA² repository:
+```
+$ git clone https://github.com/terrama2/terrama2.git .
+```
+
+1.5. Then change to the branch you want to build, if necessary:
+```
+$ git checkout -b master origin/master
+```
+
+We will assume that the codebase (all the source tree) is located at: `/home/USER/mydevel/terrama2/codebase`
+
+## Building dependencies on Linux
+
```

Comment:
There are other dependencies, check the wiki for the apt command and adding nodejs repository